runtime/lua/vim/lsp/util.lua

```diff
@@ -1388,3 +1388,4 @@ M.buf_versions = {}
 
 return M
 -- vim:sw=2 ts=2 et
+-- Test to trigger travis
```

Comment:
@runiq I guess you can drop this commit now.

Reply:
Done.